src/tryceratops/__main__.py

```diff
@@ -13,6 +13,7 @@
 runner = Runner()
 discovery = FileDiscovery()
 interface = CliInterface(runner, discovery)
+logger = logging.getLogger()
```

Comment:
When you do this, you're getting a root logger, I don't think we should do this.
Pass the name:

`__name__`

Reply:
If I add a new logger and put `logger = logging.getLogger("tryceratops")` I think that will take care of the issue